gluster-block.spec.in

```diff
@@ -97,8 +100,12 @@ rm -rf ${RPM_BUILD_ROOT}
      %attr(0755,-,-) %{_libexecdir}/gluster-block/upgrade_activities.sh
 %dir %attr(0755,-,-) %{_sharedstatedir}/gluster-block
      %attr(0644,-,-) %{_sharedstatedir}/gluster-block/gluster-block-caps.info
+%ghost %{_sharedstatedir}/gluster-block/gb_upgrade.status
 
 %changelog
+* Tue May 14 2019 Xiubo Li <xiubli@redhat.com>
+- Update the spec and fix rpmlint dangerous-command-in-%preun rm warning
```

Comment:
Do not use `%` for non-macro usage. You can escape `%` with `%%`, but I recommend to just use `rpmlint dangerous-command-in-preun rm warning`.

Also, no need to write 'Update the spec', as that is obvious ;-)

Reply:
Yeah, sure, will use "rpmlint dangerous-command-in-preun rm warning".
Thanks.